### DISCUSSION TITLE
Make Kotaemon available on local network

### DISCUSSION THREAD

**Discussion by eddprogrammer** [Cinnamon/kotaemon#568] [Q&A]
Hi, how can I run Kotaemon so other users on the same local network can access it? By default it is bind to localhost and can only be accessed locally.

**Comment by taprosoft** [comment]
@eddprogrammer just use https://www.gradio.app/guides/environment-variables#2-gradio-server-name
`export GRADIO_SERVER_NAME="0.0.0.0"`
and you are good to go.

**Comment by jake-is-ESD-protected** [comment]
**This needs to be on the main README!**
